frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Home/index.js

```
@@ -48,138 +63,974 @@ const Home = () => {
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                  padding: "20px",
+                  borderRadius: "12px",
+                  backgroundColor: "#fff",
+                  boxShadow: "0px 2px 10px rgba(0, 0, 0, 0.1)",
+                  width: "100%",
+                  position: "relative",
+                  rowGap: isMobile ? "10px" : "20px",
+                  height: isMobile ? "auto" : "150px",
+                }}
+              >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)